README.md

```diff
@@ -4,3 +4,36 @@ This repository contains ports of Tock OS (https://www.tockos.org) to Titan
 chips.
 
 This is not an officially supported Google product.
+
+
+## Getting started
+
+### Clone the repo
+
+Get the source:
+
+```shell
+git clone --recursive git@github.com:google/tock-on-titan.git
+```
+
+### Get the tools and libs to build the code
+
+Download Rust
+
+```shell
+cd tock-on-titan
+curl https://sh.rustup.rs -sSf | sh
+```
+
+```shell
+rustup default $(cat third_party/libtock-rs/rust-toolchain)
+rustup target add thumbv7m-none-eabi
+rustup default $(cat kernel/rust-toolchain)
+rustup target add thumbv7m-none-eabi
```

Comment:
It may be better to advise running `make setup` here. It is more likely to be up to date than a README. It also doesn't change the user's default rust toolchain.

Reply:
Ooh -- cool.  Will fix; will also fix osk@'s internal howto doc.

Reply:
Done.